src/wechaty/accessory_test.py

```diff
@@ -28,6 +28,10 @@ def get_user_class() -> Type[Accessory]:
 
     class FixtureClass(Accessory):
         """fixture"""
+        # fish-ball: in order to make a Accessory class properly construct,
```

Comment:
In my older refactoring, I restrict the Accessory class to be abstact, which means it should not be instantiate without a `abstract=True` field set in the derived class definition.
So the `get_user_class()` construction procedure may not match the design.
I modify the test code.